engines/sci/detection_tables.h

```diff
@@ -297,6 +297,20 @@ static const struct ADGameDescription SciGameDescriptions[] = {
 		AD_LISTEND},
 		Common::EN_ANY, Common::kPlatformDOS, 0, GUIO_STD16_UNDITHER	},
 
+	// identical to the above but on 360kb disks
```

Comment:
Please avoid referencing detection entries, because the order may change in the future. It's best to add the details of the version in this entry as well (i.e. v1.033, 6.8.90, SCI interpreter version 0.000.685)

Reply:
Fixed